components/input/theme.css

```diff
@@ -0,0 +1,280 @@
+
+  --counter-margin-top: calc(0.7 * var(--unit));
+  --counter-margin-small: calc(0.5 * var(--unit));
+  --counter-margin-large: calc(1.4 * var(--unit));
+  --counter-margin-left: calc(0.5 * var(--unit));
+  --counter-width: calc(3 * var(--unit));
+
+  --spinner-width: calc(2.9 * var(--unit));
+  --spinner-width-large: calc(3.6 * var(--unit));
+  --spinner-background: var(--color-neutral-light);
+  --spinner-border: var(--color-neutral-dark);
+  --spinner-color: var(--color-neutral-darkest);
+
+  --error-font-size: calc(1.4 * var(--unit));
+  --error-color: var(--color-ruby-dark);
+}
+
+/**
+ * Components
+ */
+.wrapper {
+  position: relative;
+
+  width: 100%;
+  max-width: var(--input-width);
+}
+
+.input {
+  transition: 0.3s ease-in-out border, 0.3s ease-in-out box-shadow;
+
+  position: relative;
+  box-sizing: border-box;
+
+  height: var(--input-height-medium);
+  width: 100%;
+  padding: 0 var(--input-horizontal-padding);
+
+  background: var(--input-background);
+
+  border: 1px solid var(--input-border-color);
+  border-radius: var(--input-border-radius);
+  box-shadow: 0;
+
+  color: var(--input-text-color);
+  line-height: var(--input-height-medium);
+  font-size: var(--input-text-size);
+  font-family: var(--font-family-regular);
+
+  &:focus {
+    border-color: var(--input-focus-border);
+    box-shadow: 0 0 0 1px var(--input-focus-border);
+  }
+
+  &:active {
+    border-color: var(--input-active-border);
+    box-shadow: inset 0 3px 6px 0 var(input-active-box-shadow);
+  }
+
+  &.is-bold {
+    font-family: var(--font-family-semi-bold);
+  }
+
+  &.is-disabled {
+    background-color: var(--input-disabled-background);
+
+    border-color: var(--input-disabled-border);
+
+    color: var(--input-disabled-color);
+  }
+
+  &.is-read-only {
+    background-color: var(--input-read-only-background);
+
+    border-color: var(--input-read-only-border);
+  }
+
+  &::placeholder {
+    color: var(--input-placeholder-color);
+  }
+}
+
+.icon {
+  position: absolute;
+  z-index: 1;
+  top: var(--icon-top);
+  left: var(--icon-horizontal);
+
+  max-height: var(--icon-height);
+
+  &,
+  svg,
+  g,
+  path {
+    fill: var(--icon-color);
+  }
+}
+
+.counter {
+  position: absolute;
+  z-index: 1;
+  top: var(--icon-top);
+  right: var(--icon-horizontal);
+}
+
+.spinner {
+  position: absolute;
+  top: 0;
+  bottom: 0;
+  right: 0;
+
+  width: var(--spinner-width);
+}
+
+.spinner-up,
+.spinner-down {
+  display: block;
+  width: 100%;
+  height: 50%;
+  margin: 0;
+
+  background-color: var(--spinner-background);
+
+  border-radius: 0 4px 0 0;
+  border: solid 1px var(--spinner-border);
+
+  &,
+  svg,
+  g,
+  path {
+    color: var(--spinner-color);
+  }
+}
+
+.spinner-down {
+  border-radius: 0 0 4px 0;
+  border-top: 0;
+}
+
+.error {
+  margin: 6px 0 0;
+
+  font-size: var(--error-font-size);
+  line-height: 1.3;
+  color: var(--error-color);
+
+  svg {
+    width: 12px;
+  }
+
+  svg,
+  g,
+  path {
+    fill: var(--error-color);
```

Comment:
FYI: You should be able to remove this (everywhere actually) when using `@teamleader/ui-icons@^0.0.7`, all you need to do is set the `color` property. 

(Check https://github.com/teamleadercrm/ui-icons/pull/3)

Reply:
That's a nice solution for this!

Reply:
You can actually remove the entire `svg { color : ... }` block, it will take the parent's color so in this case you're good to go.

Reply:
✅